src/mainboard/starlabs/labtop/variants/kbl/include/variant/hda_verb.h

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
It looks like there is a mix of spaces and tabs used for indentation.  Whilst it doesn't bother me it is likely something that the community will pick up on if / when it is upstreamed.

Reply:
#2f63c7a1db7334f7bc22c7dcca84b656f7d92ec4